### DISCUSSION TITLE
Shelly 1 Mini Gen 3 Firmware / WiFi

### DISCUSSION THREAD

**Discussion by Xcapee** [RavenSystem/esp-homekit-devices#2314] [Firmware Installation]
I was part way through working on a project some months ago and have just come back to it and am trying to figure out the current state. I believe the Shelly 1 Mini Gen 3 relay I have has been flashed to the HAA firmware, however, it is not broadcasting an HAA-XXX SSID. It also does not try to join my network, and the MAC address does not show up.

The reason I believe it already has the HAA firmware is that when I press the physical button on the relay, it toggles the light (which is not the default Shelly behaviour) and when I toggle the button 8 times, it definitely goes into another mode, which I assume is the setup mode.

But to access the setup mode I need to connect to the WiFi, and as described above, I cannot see any new advertised WiFi network, and it does not seem to have joined my network.

Is there any way with the HAA firmware to force it to enable and advertise it's default Access Point again?

**Comment by carlitzt** [comment]
Toggle the attached button 10 times fast on and off. Than the device will enter Setup mode.
Look at the connected devices in your router table, it should say something like "has-xxxxx-setup".

Then enter the IP dress of the HAA device followed by :4567 to enter the setup screen
